src/gleam/dynamic.gleam

```diff
@@ -553,7 +553,7 @@ fn at_least_decode_tuple_error(
   data: Dynamic,
 ) -> Result(a, DecodeErrors) {
   let s = case size {
-    0 -> ""
+    s if s <= 1 -> ""
```

Comment:
No need for `s`, and I think you meant to use `size` in the comparison:

```
    _ if size <= 1 -> ""
```

But you really don't even need to use the comparison since this is not a public function (i.e., you should never have a negative size):

```
    1 -> ""
```

This latter form is also consistent with what's in the rest of this file, although the other places erroneously use `0` instead of `1`, which warrants a separate issue to address.

Reply:
To be clear, I'm saying that this:

```gleam
  let s = case size {
    s if s <= 1 -> ""
    _ -> "s"
  }
```

might be better (and consistent with the existing pattern in the file) as this:

```gleam
  let s = case size {
    1 -> ""
    _ -> "s"
  }
```

so this remains exhaustive, so that even if there is a bug where size is negative, it's covered by the second clause

Reply:
@chuckwondo Thanks, I will cover it in another PR.